src/main/java/ee/buerokratt/ruuter/util/MappingUtils.java

```diff
@@ -16,11 +18,16 @@ public static JsonNode convertStringToNode(String content) {
         }
     }
 
-    public static String convertObjectToJson(Object o) {
+    public static String convertObjectToString(Object o) {
         try {
             return new ObjectMapper().writer().withDefaultPrettyPrinter().writeValueAsString(o);
         } catch (JsonProcessingException e) {
             throw new IllegalArgumentException(e);
         }
     }
+
+    public static JsonNode convertMapToNode(Map<String, Object> response) {
+        ObjectMapper mapper = new ObjectMapper();
```

Comment:
We should probalby refactor this whole class to use springs default objectmapper instead of every method creating a new objectmapper